scripts/install-fabric.sh

```diff
@@ -21,10 +21,11 @@ _arg_comp=('' )
 
 # if version not passed in, default to latest released version
 # if ca version not passed in, default to latest released version
-_arg_fabric_version="2.4.4"
-_arg_ca_version="1.5.5"
+_arg_fabric_version="2.5.0"
```

Comment:
Since the install instructions have you get install-fabric.sh from main branch, we usually abandon updating this file in the release branches.
But still, it should be the latest published release (2.4.7) so that it actually works if people try it from the release branch.

Reply:
ok